.github/workflows/ci.yml

```
@@ -195,7 +195,7 @@ jobs:
         run: cargo add --no-default-features --path ../crates/zcash_proofs
       - name: Add zcash_client_backend as a dependency of the synthetic crate
         working-directory: ./ci-build
-        run: cargo add --no-default-features --path ../crates/zcash_client_backend
+        run: cargo add --features lightwalletd-tonic-transport --path ../crates/zcash_client_backend
```

I suspect this actually needs to be what I originally said, otherwise this will just always fail, instead of checking that the old flag now passes.
```suggestion
        run: cargo add --features lightwalletd-tonic --path ../crates/zcash_client_backend
```
I've fired off a CI run to confirm this.

---

Confirmed, this fails CI as-is: https://github.com/zcash/librustzcash/actions/runs/8277319191/job/22684643018?pr=1270

---

Ah yep I misunderstood what that CI task was doing. Should have wasm builds in CI now